panacea/web/static/js/request-chain.js

```diff
@@ -0,0 +1,66 @@
+import * as View from './view';
+import * as Request from './request';
+
+export class RequestChain {
```

Comment:
if you `export default` here you can `import RequestChain from...` without the `{}`